test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/overpaymentssingle/BankAccountControllerSpec.scala

```diff
@@ -430,7 +430,7 @@ class BankAccountControllerSpec
       //        val request = FakeRequest().withFormUrlEncodedBody(form: _*)
       //        val result  = controller.enterBankAccountDetailsSubmit(journey)(request)
       //
-      //        checkIsRedirect(result, routes.ServiceUnavailableController.unavailable(journey))
+      //        checkIsRedirect(result, routes.BankAccountVerificationUnavailable.unavailable(journey))
       //      }
```

Comment:
I know it wasn't commented out in this PR but if the test is no longer valid, maybe it should just be removed

Reply:
right, these tests should be back